api/src/org/apache/cloudstack/api/command/admin/domain/ListDomainsCmd.java

```diff
@@ -53,6 +59,12 @@
                description = "If set to false, list only resources belonging to the command's caller; if set to true - list resources that the caller is authorized to see. Default value is false")
     private Boolean listAll;
 
+    @Parameter(name = ApiConstants.DETAILS,
+               type = CommandType.LIST,
+               collectionType = CommandType.STRING,
+               description = "comma separated list of host details requested, value can be a list of [ all, resource, min]")
```

Comment:
Can you clarify what would be permitted values for details parameter? If they are only "min", resource", "all" then we might want to change the name and description of this parameter to reflect true purpose. What is the difference between 'resource' and 'all'?